include/swift/AST/AutoDiff.h

```diff
@@ -329,6 +331,9 @@ class AutoDiffIndexSubset : public llvm::FoldingSetNode {
     return get(ctx, bitVec);
   }
 
+  // Use `getCapacity()` to get the total index subset capacity.
+  // This is public only for unit testing
+  // (in unittests/AST/SILAutoDiffIndices.cpp).
```

Comment:
Hmm how? I don't see any unit tests added in this patch.

Reply:
`unittests/AST/SILAutoDiffIndices.cpp` wasn't added in this patch.